### DISCUSSION TITLE
a suggestion to make a descent whiteboard for basic note taking.

### DISCUSSION THREAD

**Discussion by mylastresort** [lawrencehook/remove-youtube-suggestions#102] [Ideas]
it would be really helpful and excellent idea to make a simple whiteboard or a integrating famous whiteboard app if possible, when the contents is centered and there is enough room to hold it.
As a student of 42 network, we rely occasionally on video content, it would be very nice to have it.

With much respect to the team behind this project.

**Comment by lawrencehook** [comment]
Thanks for the comment! I have another extension called [WebStickies](https://lawrencehook.com/ws) that you might find useful.

**Comment by mylastresort** [comment]
Thanks a lot, this is quite helpful!
